server/sublist_test.go

```diff
@@ -1047,9 +1048,14 @@ func TestSublistAll(t *testing.T) {
 var benchSublistSubs []*subscription
 var benchSublistSl = NewSublistWithCache()
 
+// https://github.com/golang/go/issues/31859
```

Comment:
I know we're not yet using go1.13 but I happened to run tests on a go1.13 machine and it all failed outright so this is a fix for that and appears to work on 1.12 too